utils/docker/Dockerfile.centos.7

```diff
@@ -118,6 +118,21 @@ ARG QUICKBUILD_DEPS=""
 ARG REPOS=""
 ARG REPO_URL=""
 ARG REPO_EL7=""
+# Need to clean this up later
+RUN if [ -n "$REPO_URL" ]; then                                             \
+      cversion=$(cut -d ' ' -f4 /etc/centos-release);                       \
+      version=${cversion%.*} ;                                              \
+      MY_REPO="${REPO_URL}repository/centos-${version}-x86_64-group/";      \
+      MY_NAME="${MY_REPO#*//}";                                             \
+      MY_NAME="${MY_NAME//\//_}";                                           \
+      echo -e "[${MY_NAME}]\n\
+name=created from ${MY_REPO}\n\
+baseurl=${MY_REPO}\n\
+enabled=1\n\
+gpgcheck=False\n" >> /etc/yum.repos.d/local-centos-group.repo;              \
+      yum --assumeyes install yum-utils;                                    \
```

Comment:
Should fix these to align the continuator with the longest line below.

Reply:
Done in next push

Reply:
Ahhh.  You made the longest line shorter by breaking it into two lines.  You could pull back all of the continuators now, but that's not terribly important.